app/services/market_history_dedupe_service.rb

```diff
@@ -34,6 +34,7 @@ def dedupe(data, server_id)
     if REDIS[server_id].get("HISTORY_RECORD_SHA256:#{sha256}").nil?
       REDIS[server_id].set("HISTORY_RECORD_SHA256:#{sha256}", '1', ex: 600)
       return if data['AlbionId'] == 0 # sometimes the client sends us 0 for the numeric item id, we trash this data
```

Comment:
remove this, not needed since it's in your || statement

Reply:
This one is testing AlbionId == 0. The changes I made test LocationId. I kept it.